solution-base/mongodb/charts/mongodb-sharded/values.yaml

```diff
@@ -353,7 +353,9 @@ configsvr:
   ## mongodbExtraFlags:
   ##  - "--wiredTigerCacheSizeGB=2"
   ##
-  mongodbExtraFlags: []
+  mongodbExtraFlags:
+    - "--setParameter rollbackTimeLimitSecs=259200"
+
```

Comment:
I think we set many parameters through the makefile as well : any specific reason we set this here instead?

Reply:
Because the Makefile for now does not work anymore. Charts are not stored at the same place and they put in place a 6 month retention policy AFAIK. I am planning to work on it with the 4.4 upgrade.

I can add it to a patch file though, so that we don't loose this change later.

Reply:
> Because the Makefile for now does not work anymore. Charts are not stored at the same place and they put in place a 6 month retention policy AFAIK. I am planning to work on it with the 4.4 upgrade.

i meant the makefile used to render the chart and generate the Zenko-Base ISO, which does not depend on the url or 6-month retention, but just renders the chart with some specific values :

* https://github.com/scality/Zenko/blob/40ea563f558a4f75f3c50ea1415784c91091c12a/solution-base/build.sh#L128
* https://github.com/scality/Zenko/blob/40ea563f558a4f75f3c50ea1415784c91091c12a/solution-base/build.sh#L166

(looks like it is a build.sh script and not a makefile, sorry for the confusion)

Reply:
Oh ok I see, then I will make the appropriate changes :smile: 

I also foudn the workaround for the Makefile, putting it here but I'll do it later:

`CHART_REPO:="https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami"`

Source: https://github.com/bitnami/charts/issues/10539

Reply:
Done (and rebased) in https://github.com/scality/Zenko/pull/1870/commits/456d0a1acec6975ac62f58ab7ba9fdbcb6b471dc

Reply:
Update after some rebases: done in https://github.com/scality/Zenko/pull/1870/commits/3fcc921411eb427a64a601120ed1dba054a41cf3 for the sharded mode only. I do not want to ointroduce more entropy in this PR, and the fix is only for an Artesca use case, so sharded.